pkgs/by-name/lu/luastatus/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+# Native Build Inputs
+, cmake
+, pkg-config
+, makeWrapper
+# Dependencies
+, xorg
+, yajl
+, alsa-lib
+, libpulseaudio
+, glib
+, libnl
+, udev
+, libXau
+, libXdmcp
+, pcre2
+, pcre
+, util-linux
+, libselinux
+, libsepol
+, lua5
+, docutils
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "luastatus";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "shdown";
+    repo = "luastatus";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-whO5pjUPaCwEb2GDCIPnTk39MejSQOoRRQ5kdYEQ0Pc=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    yajl
+    alsa-lib
+    libpulseaudio
+    glib
+    libnl
+    udev
+    pcre2
+    pcre
+    util-linux
+    libselinux
+    libsepol
+    lua5
+    docutils
+  ] ++ (with xorg; [ libxcb libX11 xcbutil xcbutilwm libXdmcp libXau ]);
+
+  postInstall = ''
+    wrapProgram $out/bin/luastatus-stdout-wrapper \
+      --prefix LUASTATUS : $out/bin/luastatus
+
+    wrapProgram $out/bin/luastatus-i3-wrapper \
+      --prefix LUASTATUS : $out/bin/luastatus
+
+    wrapProgram $out/bin/luastatus-lemonbar-launcher \
+      --prefix LUASTATUS : $out/bin/luastatus
+  '';
+
+  meta = with lib; {
+    description = "Universal status bar content generator";
```

Comment:
Not sure, but for homepage I think it's legit to reuse the version.
`homepage = "https://github.com/shdown/luastatus/releases/tag/v${finalAttrs.version}";`

Reply:
Whoops, this field was actually supposed to be `changelog` with that as it's value.

Fixed in https://github.com/NixOS/nixpkgs/pull/256033/commits/b951cae739e573385e02458354159ce2c4bac2a1